### PR TITLE
Hotfix : change method to validate mimetypes

### PR DESCRIPTION
**Pull request by fhfournier**

### CODE REVIEW

File: upload.php [hunk 1]
```diff
@@ -222,17 +222,16 @@ function emr_normalize_file_urls( $old, $new ) {
...
-	if ($filedata["ext"] == "") {
+	if ( empty( $filedata["ext"] ) || !array_key_exists( $filedata["ext"], get_allowed_mime_types() ) ) {
 		echo esc_html__("File type does not meet security guidelines. Try another.", 'enable-media-replace');
 		exit;
 	}
 
 	$new_filename = $_FILES["userfile"]["name"];
 	$new_filesize = $_FILES["userfile"]["size"];
-	$new_filetype = $filedata["type"];
+	$new_filetype = $filedata[ "ext" ];
```
Comment (short-pixel-optimizer) [inline upload.php]: Thanks for your contribution. Why did you change this line? It looks that the $new_filetype needs to be a mime-type (for example image/png) further on and not just the (png) extension, but maybe I'm wrong.

Comment (fhfournier) [reply upload.php]: (my bad) Good question! lol i think its for the condition under this line. I made this quick update for our customer and everthing work. But i think you should validate the condition after and maybe rename variable.

**Comment by fhfournier** [comment]
Line 238 : i'm not shure if you want to get the file extension or mimetype

Im not shure this line its a good solution and maybe use only what $filedata have inside of it!
